eng/pipelines/templates/scripts/create-tag-for-current-branch.ps1

```diff
@@ -0,0 +1,26 @@
+<#
```

Comment:
Why is this under here as opposed to under eng/scripts like the other scripts?

Reply:
The template which calls the script is under eng/pipelines.
That's why I am thinking to have the script somewhere under this folder `eng/pipelines`.
 Is there any principal I can follow here?

Reply:
In general we try to keep things under pipelines related to yml file for devops pipelines and we put the scripts under the scripts folder parallel to the pipelines folder under eng.